core/src/main/scala/cats/data/ContT.scala

```diff
@@ -47,6 +47,8 @@ sealed abstract class ContT[M[_], A, +B] extends Serializable {
       M.defer(run(fn3))
     }
   }
+
+  final def eval(implicit M: Applicative[M], D: Defer[M], ev: B <:< A): M[A] = run(b => D.defer(M.pure(ev(b))))
```

Comment:
I think we need instead:
```scala
D.defer(run(b => M.pure(ev(b))))
```
as you have it now, it is not needed, since `b` is already evaluated, so `D.defer(M.pure(ev(b)))` is not helpful.

Reply:
Yes, good point! 👍